pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx

```diff
@@ -67,58 +68,15 @@ class PokemonDisplay extends React.Component<displayProps, displayState> {
   }
 
   render() {
-    let pokemon = this.pokemonToDisplay;
-    let pokemonName: string | undefined;
-    let dexId: string;
-    let displayDefault: string | undefined = "";
-    let displayShiny: string | undefined = "";
-    let type1: string | null;
-    let type2: string | null;
-    let stats: {
-      hp: number;
-      attack: number;
-      defense: number;
-      spAttack: number;
-      spDefense: number;
-      speed: number;
-    } | null;
-    let baseStats: string;
+    const pokemon = this.pokemonToDisplay;
 
     if (pokemon) {
-      pokemonName = pokemon.name;
-      dexId = `#${pokemon.dexId}`;
-      displayDefault = pokemon.frontDefault;
-      displayShiny = pokemon.frontShiny;
-      type1 = pokemon.type1;
-      type2 = pokemon.type2 ? pokemon.type2 : null;
-      stats = pokemon.stats;
-      baseStats = "Total base stats: " + pokemon.baseStats.toString();
-
       return (
         <Container maxWidth="sm">
-          <QuickView
-            pokemonName = {pokemonName} 
-            dexId = {dexId} 
-            baseStats = {baseStats}
-            stats = {stats}
-            type1 = {type1} 
-            type2 = {type2} 
-            displayDefault = {displayDefault} 
-            displayDefaultS = {displayShiny}
-          />
+          <QuickView pokemon={pokemon} />
```

Comment:
The `<QuickView>` no longer has a specific set of inputs, it just renders a `PokemonDTO`.